.github/scripts/js/ci.js

```diff
@@ -779,17 +896,63 @@ const findAndRerunWorkflow = async ({ github, context, core, workflow_id }) => {
  */
 module.exports.createReleaseIssueForMilestone = async ({ github, context, core }) => {
   const milestone = context.payload.milestone;
+
+  const found = milestone.title.match(/(v\d+\.\d+\.\d+)/);
+  if (!found) {
+    return core.setFailed(`Milestone '${milestone.title}' not dedicated to release version in form of vX.Y.Z. Ignore creating release issue.'`);
+  }
+  const milestoneVersion = found[1];
+
   // NOTE: non-breaking space after emoji.
-  const issueBody = `:point_right: Use this issue to test milestone [${milestone.title}](${milestone.html_url}) and deploy released tag.
-            :point_right: Use 'e2e/run/' labels to run default e2e test.
-            :point_right: Use 'e2e/use/' labels to run specific e2e test.
-            :point_right: Use 'deploy/' labels to deploy site and documentation.
-            :point_right: Use 'deploy/deckhouse/' labels to deploy to channels after creating tag.`;
+  const issueBody = `:robot: A dedicated issue to run tests and deploy release [${milestoneVersion}](${milestone.html_url}).
+
+---
+
+<details>
+<summary>Release issue commands and options</summary>
+<br />
+
+You can trigger release actions by commenting on this issue:
+
+- \`/deploy/<channel>\` will publish built images into the release channel.
```

Comment:
I assume that `/deploy/alpha/ce` is a valid command, but there is no info in this FAQ about editions.

Reply:
Add documentation about editions.